Index: src code/lab2/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src code/lab2/.idea/.gitignore b/src code/lab2/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1776541963248)
+++ b/src code/lab2/.idea/.gitignore	(date 1776541963248)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: src code/lab2/.idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src code/lab2/.idea/modules.xml b/src code/lab2/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1776541963248)
+++ b/src code/lab2/.idea/modules.xml	(date 1776541963248)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/lab2.iml" filepath="$PROJECT_DIR$/lab2.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: src code/lab2/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src code/lab2/.idea/vcs.xml b/src code/lab2/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1776541963246)
+++ b/src code/lab2/.idea/vcs.xml	(date 1776541963246)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/../../.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: src code/lab2/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src code/lab2/.idea/misc.xml b/src code/lab2/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1776541963246)
+++ b/src code/lab2/.idea/misc.xml	(date 1776541963246)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
diff --git a/src code/lab2/Aims.java b/src code/hust/soict/dsai/aims/Aims.java
rename from src code/lab2/Aims.java
rename to src code/hust/soict/dsai/aims/Aims.java
diff --git a/src code/lab2/Cart.java b/src code/hust/soict/dsai/aims/cart/Cart.java
rename from src code/lab2/Cart.java
rename to src code/hust/soict/dsai/aims/cart/Cart.java
diff --git a/src code/lab2/DigitalVideoDisc.java b/src code/hust/soict/dsai/aims/disc/DigitalVideoDisc.java
rename from src code/lab2/DigitalVideoDisc.java
rename to src code/hust/soict/dsai/aims/disc/DigitalVideoDisc.java
